src/main/java/uml/memo/service/UmlService.java

```diff
@@ -48,13 +49,17 @@ public String decode(String encoded) throws IOException {
         }
     }
 
-    public byte[] decodeImageWithCache(String encodedUml, FileFormat format) throws IOException, MemcachedService.Exception {
+    public String decodePlantUmlString(String encoded) throws IOException {
+        return TranscoderUtil.getDefaultTranscoder().decode(encoded);
+    }
+
+    public byte[] decodeImageWithCache(String encodedUml, FileFormat format, boolean isPlantUmlEncode) throws IOException, MemcachedService.Exception {
         String key = "uml:" + format.name() + ":" + DigestUtils.md5DigestAsHex(encodedUml.getBytes(StandardCharsets.UTF_8));
```

Comment:
Is it safe to not include `isPlantUmlEncode` in this cache key?

Reply:
The generated key of uml of this pull request description is
`uml-memo:uml:SVG:true:10486d64889478be12b6cf322abadc88`.